<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistColorLibForm</class>
 <widget class="QWidget" name="RegistColorLibForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frameRegistPanel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>204</width>
     <height>444</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>431</width>
     <height>411</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="RGB">
    <widget class="QFrame" name="frameRGBPanel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>420</width>
       <height>356</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ColorSpace">
    <widget class="QFrame" name="frameColorPanel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>420</width>
       <height>356</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>閉じる</string>
   </property>
   <property name="icon">
    <iconset resource="DisplayImage.qrc">
     <normaloff>:/Qtres/Close.png</normaloff>:/Qtres/Close.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="DisplayImage.qrc"/>
 </resources>
 <connections/>
</ui>
